tcms/testruns/static/testruns/js/get.js

```diff
@@ -86,9 +83,59 @@ function renderProgressBars(positiveCount, negativeCount, allCount) {
 
 function renderCountPerStatusList(statusCount) {
     for (var status in statusCount) {
-        const status_id = statusCount[status].id;
+        const statusId = statusCount[status].id;
 
-        $(`#count-for-status-${status_id}`).attr('href', `?status_id=${status_id}`);
-        $(`#count-for-status-${status_id}`).text(statusCount[status].count);
+        $(`#count-for-status-${statusId}`).attr('href', `?status_id=${statusId}`).text(statusCount[status].count);
     }
 }
+
+function renderTestExecutions(testExecutions, executionStatuses) {
+    const container = $('#test-executions-container')
+    const testExecutionRowTemplate = $('#test-execution-row')[0].content
+
+    const testExecutionCaseIds = []
+    testExecutions.forEach(testExecution => {
+        testExecutionCaseIds.push(testExecution.case_id)
+
+        const executionStatus = executionStatuses.find(status => status.id === testExecution.status_id)
+        const template = $(testExecutionRowTemplate.cloneNode(true))
+
+        container.append(renderTestExecutionRow(template, testExecution, executionStatus))
+    })
+
+    treeViewBind();
+    renderAdditionalInformation(testExecutions, testExecutionCaseIds)
+}
+
+function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
+    jsonRPC('TestCase.filter', { 'id__in': testExecutionCaseIds }, testCases => {
+        testExecutions.forEach(testExecution => {
+            const testCase = testCases.find(testCase => testCase.id === testExecution.case_id)
+
+            const listGroupItem = $(`.test-execution-${testExecution.id}`)
+            const automatedIcon = testCase.is_automated ? 'fa-thumbs-up' : 'fa-cog'
+            listGroupItem.find('.test-execution-automated').addClass(automatedIcon)
+            listGroupItem.find('.test-execution-priority').html(testCase.priority)
+            listGroupItem.find('.test-execution-category').html(testCase.category)
```

Comment:
Missing title for auto/manual but the rest of the attributes are there. See the new TP page.